pandas/io/formats/style_render.py

```diff
@@ -366,9 +366,11 @@ def _translate(
         if not get_option("styler.html.mathjax"):
             table_attr = table_attr or ""
             if 'class="' in table_attr:
-                table_attr = table_attr.replace('class="', 'class="tex2jax_ignore ')
+                table_attr = table_attr.replace(
+                    'class="', 'class="tex2jax_ignore mathjax_ignore"'
```

Comment:
This should be

    'class="', 'class="tex2jax_ignore mathjax_ignore '

as we are prepending to an already existing class.